feat: add docs for LogRocket and Google OAuth (#385)
chatwoot/docs: docs/self-hosted/configuration/environment-variables.md

```bash
DIRECT_UPLOADS_ENABLED=true
```

## Google OAuth

To enable Google OAuth in Chatwoot, you need to provide the client ID, client secret, and callback URL. You can find the instructions to generate the details [here](https://support.google.com/cloud/answer/6158849).

Set the `GOOGLE_OAUTH_CLIENT_ID` and `GOOGLE_OAUTH_CLIENT_SECRET` environment variables in your Chatwoot installation using the values you copied from the Google API Console. Set the `GOOGLE_OAUTH_CALLBACK_URL` environment variable to the callback URL you used in the Google API Console. Here's an example of the same

```bash
GOOGLE_OAUTH_CLIENT_ID=369777777777-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.apps.googleusercontent.com
GOOGLE_OAUTH_CLIENT_SECRET=ABCDEF-GHijklmnoPqrstuvwX-yz1234567
GOOGLE_OAUTH_CALLBACK_URL=https://<your-server-domain>/omniauth/google_oauth2/callback
```

> **_The callback url should comply with the format in the example above. This endpoint cannot be changed at the moment._**

Reviewer: ```suggestion
**_The callback URL should comply with the format in the example above. This endpoint cannot be changed at the moment._**
```

After setting these environment variables, restart your Chatwoot server to apply the changes. Now, users will be able to sign in using their Google accounts.

## LogRocket

> To enable LogRocket in Chatwoot, you need to provide the project ID. Here are the steps to follow:

Reviewer: ```suggestion
To enable LogRocket in Chatwoot, you need to provide the project ID. Follow the steps as described below.
```

> Go to the LogRocket [website](https://logrocket.com/) and create an account or sign in to your existing account. After signing in, create a new project in LogRocket by clicking on the "Create new project", enter a name for your project, and get the project ID.

Reviewer: ```suggestion
**Step 1.** Open the LogRocket [website](https://logrocket.com/) and create an account or sign in to your existing account. 

**Step 2.** After signing in, create a new project in LogRocket by clicking on "Create new project". 

**Step 3.** Enter a name for your project, and save the project ID.
```

> Set the `LOG_ROCKET_PROJECT_ID` environment variable in your Chatwoot installation using the project ID you copied from LogRocket.

Reviewer: ```suggestion
**Step 4.** Set the `LOG_ROCKET_PROJECT_ID` environment variable in your Chatwoot installation using the project ID you copied from LogRocket.
```
